inc/options/class-wpseo-options.php

```diff
@@ -168,28 +168,57 @@ public static function get_option_names() {
 	 * well change between calls (enriched defaults and such)
 	 *
 	 * @static
-	 * @return  array  Array combining the values of (nearly) all the options
+	 * @return  array  Array combining the values of all the options
 	 */
 	public static function get_all() {
```

Comment:
You are using this method only in one test. Please change the call of that into: `WPSEO_Options::get_options( self::get_option_names() );`